release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2548,6 +2548,15 @@ Space precluded documenting all of the container images for this release in the
 
 link:https://access.redhat.com/solutions/5590681[{product-title} 4.6.6 container image list]
 
+[id="ocp-4-6-6-bug-fixes"]
+==== Bug fixes
+
+* Prior to {product-title} 4.6, when the Marketplace Operator used the `OperatorSource` custom resource definition (CRD), cluster administrators using a cluster on a restricted network (also known as a disconnected cluster) could disable the default `OperatorSource` objects in the `openshift-marketplace` namespace and create custom `CatalogSource` objects with the same name as the default sources. In {product-title} 4.6, the Marketplace Operator uses `CatalogSource` objects directly now that the `OperatorSource` CRD is removed. As a result, the `openshift-marketplace` had default catalog sources that were managed by the OperatorHub API.
++
+After disabling the default catalog sources on a disconnected {product-title} 4.6 cluster, when an administrator tried to create a catalog source with the same name as that of the default sources, the OperatorHub API previously removed the custom catalog source. If the catalog source was not disabled using the OperatorHub API and changes were made to the default catalog source (for example changing the `spec.image` parameter to point to an internal registry for the disconnected environment), the spec was restored to the default spec.
++
+This bug fix allows for the creation, update, and deletion of custom catalog sources with same name as the default sources, if they are disabled using the OperatorHub API. As a result, administrators can now disable the default catalog sources and create custom catalog sources using the default names without them being removed or overwritten. If the default catalog source is re-enabled, the default spec is restored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1895952[*BZ#1895952*])
```

Comment:
I think s/update/updating for tense matching in the first sentence here. But it might sound sort of awkward even when changed, so perhaps make the subject "you" for less taxing reading? i.e.:
> This bug fix allows you to create, update, and delete custom catalog sources with same name as the default sources, if they are disabled using the OperatorHub API.

(Or "allows administrators" if that's more accurate?)

Reply:
Went w/ the latter!